app/Console/Commands/AdsBroadcastHost.php

```diff
@@ -64,36 +57,16 @@ public function __construct()
      *
      * @return int
      */
-    public function handle(AdsClient $adsClient): int
+    public function handle(AdsClient $adsClient): void
     {
         $this->info('Start command '.$this->signature);
 
-        $message = $this->strToHex(urlencode(self::BROADCAST_PREFIX.$this->infoApiUrl));
-
-        $command = new BroadcastCommand($message);
-        try {
-            $response = $adsClient->runTransaction($command);
-            $txid = $response->getTx()->getId();
-            $this->info("Broadcast message sent successfully. Txid: [$txid]");
-        } catch (CommandException $exc) {
-            $this->error('Cannot send broadcast due to error '.$exc->getCode());
+        $command = new BroadcastCommand(new Url($this->infoApiUrl));
```

Comment:
Actually, this could go even further, to create `InfoUrlBroadcastMessage` which would be `Hexable` and would have a method to `->getInfoUrl(): Url` so we could use the same object in Broadcast Handler. 
Maybe...